From 88f0ba789a24973785444f18dc22e91ab38ea79e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=83=C2=B3=20Gy=C3=83=C2=B6rgy?= <ballogyor@gmail.com>
Date: Thu, 18 Jan 2018 19:41:32 +0100
Subject: [PATCH] Fix build when gtkspell enabled

HAVE_GTKSPELL needs to be defined, otherwise the gtkspell related code never built.
---
 meson.build     | 1 +
 src/meson.build | 4 ++++
 2 files changed, 5 insertions(+)

diff --git a/meson.build b/meson.build
index 00f3b2ab..c20f8a6f 100644
--- a/meson.build
+++ b/meson.build
@@ -125,6 +125,7 @@ endif
 
 if get_option('enable-gtkspell')
   gtkspell = dependency('gtkspell3-3.0')
+  conf.set('HAVE_GTKSPELL', 1)
 endif
 
 if get_option('enable-polkit')
diff --git a/src/meson.build b/src/meson.build
index bf189241..c90366b3 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -100,6 +100,10 @@ if get_option('enable-gnome-desktop')
   gnome_software_dependencies += [gnome_desktop]
 endif
 
+if get_option('enable-gtkspell')
+  gnome_software_dependencies += [gtkspell]
+endif
+
 executable(
   'gnome-software',
   resources_src,
-- 
2.15.1

